draft-ietf-rats-corim.md

```diff
@@ -1171,7 +1177,15 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/310
+A Conditional Endorsement Series triple uses a stateful environment, (i.e. `stateful-environment-record`), that identifies a Target Environment based on an `environment-map` plus the `measurement-map` measurements that have matching Evidence.	
+
+The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
+Each conditional-series-record record is evaluated in the order it appears in the series array.
+The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
```

Comment:
This one too seems too deep too soon: reference to internals, mention of ACS.

Reply:
We can change ACS to attester's actual state